### PR TITLE
Release: 13.0.0 

### PR DESCRIPTION
**Pull request by ahmedAlaaInstabug**
## Description of the change
Release: v13.0.0

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
## Related issues

Jira ID : MOB-14181

### Development
- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
### Code review 
- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## [Unreleased](https://github.com/Instabug/Instabug-React-Native/compare/v12.7.0...dev)
+## [13.0.0](https://github.com/Instabug/Instabug-React-Native/compare/v12.7.0...dev) (April 28, 2024)
```
Comment (a7medev) [inline CHANGELOG.md:3]: I think this should now be April 29 😅
```suggestion
## [13.0.0](https://github.com/Instabug/Instabug-React-Native/compare/v12.7.0...dev) (April 29, 2024)
```

File: e2e/Utils/CaptainTest.cs [hunk 2]
```diff
@@ -9,7 +9,7 @@ public class CaptainTest : IDisposable
   {
     AndroidApp = Path.GetFullPath("../../../../example/build/app/outputs/flutter-apk/app-debug.apk"),
     AndroidAppId = "com.instabug.flutter.example",
-    AndroidVersion = "14",
+    AndroidVersion = "13",
```
Comment (a7medev) [inline e2e/Utils/CaptainTest.cs:12]: Why do we need to change the Android emulator version in this PR?

**Review by a7medev — CHANGES_REQUESTED** [review]

File: e2e/Utils/CaptainTest.cs (hunk 2 above)
Comment (ahmedAlaaInstabug) [reply e2e/Utils/CaptainTest.cs:12]: since circle-ci uses Android 13 ,, so Android version 14 was a mistake :D

**Review by a7medev — APPROVED** [review]
LGTM 👌🏼

**Pull request merged**